### PR TITLE
Add starter around RSC Actions

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: apps/starter-nextjs-rsc-actions/components/waitlist.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    await xata.db.emails.create({ email });
+  }
+}
+
+export default function Waitlist() {
+  return (
+    <form action={create} className="flex gap-6 mb-6">
+      <input
+        type="email"
+        id="email"
```
Comment (atilafassina) [inline apps/starter-nextjs-rsc-actions/components/waitlist.tsx:18]:
```suggestion
        name="email"
```

**Comment by snide** [comment]
@SferaDev I added the following:

- A minimal success page
- Better readme instructions
- Cleaned up all the broken links
- Made it so you can load from a schema

**Comment by alexfrancoeur** [comment]
Following up here, I know there are a number of competing priorities. When are we thinking we might be able to get this over the line?

**Comment by atilafassina** [comment]
> Following up here, I know there are a number of competing priorities. When are we thinking we might be able to get this over the line?

I can do this.
First step will be to move this into its own repo, so I'll ask to freeze work on this branch.

**Pull request closed without merging**